module_utils/network_lsr/argument_validator.py

```diff
@@ -465,6 +475,22 @@ def _validate_post(self, value, name, result):
 
 
 class ArgValidator_DictIP(ArgValidatorDict):
+    VALID_MODES = [
+        "debug",
+        "ndots:\d+",
```

Comment:
Please add unit tests for that show that valid values are accepted and invalid values are not accepted.

Reply:
Hi Till, you mean the python unit test file - `test_network_connections.py`?

Reply:
yes